### ISSUE TITLE
Performance for fetching a file from nexus very poor

### ISSUE THREAD

**Issue by alex4200** [BlueBrain/nexus-python-sdk#77]
When downloading a 100 MB file through the nexus SDK using `files.fetch` the performance is very weak. It takes almost 3 minutes to download the file, as opposed to 1 second(!) using curl. 

Actual command:

```
nexus.files.fetch(org_label=config.ORG, project_label=config.PROJECT, file_id = file_id,out_filepath = "out.dat")
```

with ORG/PROJECT = `gpfs_tests`, `file_id="http://dev.nexus.ocp.bbp.epfl.ch/v1/resources/gpfs_tests/gpfs_tests/_/e16577f8-9f64-45e0-9623-cca82bed8644"` 

Version used of nexussdk: commit 8c8516f1c7d53eea89754ab0ffe2bd9f9e6f07c9

**curl command:**

`curl -H 'Cache-Control: no-cache'  -H "Authorization: Bearer ${TOKEN}"  -XGET "$FILEID"  -o data.dat`

**Comment by alex4200** [comment]
This issue seems to be back in the staging environment. 

100MB file with ID: `https://staging.nexus.ocp.bbp.epfl.ch/v1/resources/gpfs_tests/gpfs_tests/_/8b071a0d-8223-4153-9b96-b7a2ba5cf5c3`

- using SDK it takes about 3 minutes to download the file
- with curl it is about 2 seconds: `curl -H 'Cache-Control: no-cache' -H "Authorization: Bearer ${TOKEN}" -XGET "https://staging.nexus.ocp.bbp.epfl.ch/v1/resources/gpfs_tests/gpfs_tests/_/8b071a0d-8223-4153-9b96-b7a2ba5cf5c3" -o test.dat`

**Comment by hygt** [comment]
@alex4200 I've published a new version `0.3.2` on PyPI that should fix the issue.
